src/utils/sources-tree/addToTree.js

```diff
@@ -86,8 +86,9 @@ function traverseTree(
   tree: TreeDirectory,
   debuggeeHost: ?string
 ): TreeNode {
-  const parts = url.path.split("/").filter(p => p !== "");
-  parts.unshift(url.group);
+  const parts = url.path.split("/").filter((p, i) => p !== "" || i < 2);
```

Comment:
We should add a comment here as to what is going on.

Reply:
I agree that this filter seems unnecessary to me. My vote would be for

```
const parts = url.path.split("/");
if (parts[0] === "") parts.shift();
parts.unshift(url.group);
```

Reply:
Looking over this again, I think we should just keep it simple and do 
```
const parts = url.path.split("/");
parts.unshift(url.group);
```
and revisit after that.

Reply:
@loganfsmyth Unfortunately the snippet you cited creates the following:

<img width="298" alt="tree" src="https://user-images.githubusercontent.com/46655/52297005-5bbb6200-2945-11e9-9043-e1784ff57a82.png">

Reply:
Alright, that's what I was worried about. Honestly, maybe the better approach would be to just leave the filter for now and change things more in a later pass?

Reply:
@darkwing  in davidwalsh.name, the empty folder you got was because of a path '/', which creates empty node. for this case, we can remove any trailing slash from a url.

otherwise this works fine. can you please have a look at my commit.